### PR TITLE
Split test suites for H265 and H266

### PR DESCRIPTION
**Pull request by rsanchez87**

### CODE REVIEW

File: scripts/gen_jct_vc.py
Comment (mdimopoulos) [inline scripts/gen_jct_vc.py:1]: This file contains code that you removed in https://github.com/fluendo/fluster/pull/177, e.g. lines 108-110. There are other cases, please check

Also there is H266 related code that should be removed, e.g. line 38, lines 291-298. Please do a careful pass

File: scripts/gen_jvet.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            data = str(resp.read())
+            hparser.feed(data)
+
+        for url in hparser.links[1:]:
+            # The first item in the AVCv1 list is a readme file
+            if "00readme_H" in url:
+                continue
+            elif "replaced" in url:
+                # This is in HEVC-SHVC, we don't want that.
+                continue
```
Comment (mdimopoulos) [inline scripts/gen_jvet.py:109]: To remove

File: scripts/gen_jvet.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+                    test_suite.resources_dir, test_suite.name, test_vector.name
+                )
+                + os.sep,
+                "",
+            )
+            if not test_vector.input_file:
+                raise Exception(f"Bitstream file not found in {dest_dir}")
+            test_vector.source_checksum = utils.file_checksum(dest_path)
+            if "main10" in test_vector.name.lower():
+                test_vector.output_format = OutputFormat.YUV420P10LE
```
Comment (mdimopoulos) [inline scripts/gen_jvet.py:143]: I think this can be removed for H266

File: scripts/gen_jvt.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # Fluster - testing framework for decoders conformance
-# Copyright (C) 2020, Fluendo, S.A.
+# Copyright (C) 2024, Fluendo, S.A.
```
Comment (mdimopoulos) [inline scripts/gen_jvt.py:4]: Check this out https://stackoverflow.com/questions/9039698/which-years-to-write-into-copyright-header.
The most common way for the year in copyright is one of
- first year that this code was worked on, e.g. 2020
- first to last year interval, e,g. 2020-2024

the reason for this is that although the file is new, the code was inherited and modified based on a previous implementation

File: scripts/gen_jvt.py [hunk 4]
```diff
@@ -37,7 +37,6 @@
 BASE_URL = "https://www.itu.int/"
 H264_URL = BASE_URL + "wftp3/av-arch/jvt-site/draft_conformance/"
 BITSTREAM_EXTS = (
-    ".bit",
```
Comment (mdimopoulos) [inline scripts/gen_jvt.py:40]: Don't we have any .bit files in H264 test suites?

**Review by mdimopoulos — CHANGES_REQUESTED** [review]

File: scripts/gen_jvt.py (hunk 4 above)
Comment (rsanchez87) [reply scripts/gen_jvt.py:40]: No... I forgot remove it (in your comment) in the last PR:

https://github.com/fluendo/fluster/pull/177

File: scripts/gen_jvt.py (hunk 3 above)
Comment (rsanchez87) [reply scripts/gen_jvt.py:4]: Done

File: scripts/gen_jvet.py (hunk 2 above)
Comment (rsanchez87) [reply scripts/gen_jvet.py:143]: Hmm, yes, Done, thanks

File: scripts/gen_jct_vc.py
Comment (rsanchez87) [reply scripts/gen_jct_vc.py:1]: Thanks again, I mixed badly the saved files previously with last files in the previous PR:

 https://github.com/fluendo/fluster/pull/177

Comment (mdimopoulos) [reply scripts/gen_jct_vc.py:1]: Lines 108-110 are still there :). Can you take a look please?

Comment (mdimopoulos) [reply scripts/gen_jct_vc.py:1]: Lines 216, 218 do not need to check for the codec anymore

**Review by mdimopoulos — CHANGES_REQUESTED** [review]

File: scripts/gen_jct_vc.py
Comment (rsanchez87) [reply scripts/gen_jct_vc.py:1]: I didn't understand very well the comment, sorry again... done.

**Review by mdimopoulos — APPROVED** [review]

**Pull request merged**